config/yamlcfg/yaml2proto_test.go

```diff
@@ -710,3 +712,103 @@ func Test_ReadConfig(t *testing.T) {
 		})
 	}
 }
+
+func Test_getDefaults(t *testing.T) {
+	tests := []struct {
+		name  string
+		paths []string
+		want  []string
+	}{
+		{
+			name: "empty paths",
+		},
+		{
+			name:  "simple case",
+			paths: []string{"foo/config.yaml", "foo/default.yaml"},
+			want:  []string{"foo/default.yaml"},
+		},
+		{
+			name:  "no defaults",
+			paths: []string{"foo/config.yaml"},
+		},
+		{
+			name:  "multiple defaults",
```

Comment:
What happens if `default.yaml` and `default.yml` exist together?

Reply:
Done